modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -29,12 +29,40 @@ Use this option if you created your API outside API Manager, but want to apply p
 Use this option if you want to provide additional security for your API. 
 +
 . Based on the type of endpoint you are creating, specify the application type for a basic endpoint or the proxy deployment target for an endpoint with proxy.
-. If you want to manage the API using Mule 4 or later, select the *Mule version* checkbox.
-> In *Implementation URI*, replace the example by entering the URL of the site that hosts the API.
+. If you want to manage the API using Mule 4 or later, perform the following steps:
+.. Select the *Mule version* checkbox.
+.. In *Implementation URI*, replace the example by entering the URL of the site that hosts the API.
 +
 For example, you can use the URL of your API asset in Exchange. In this case, the API is consuming the JSONPlaceholder API, so enter `+http://jsonplaceholder.typicode.com+`.
+.. If you want to secure outbound traffic, click the elipses (*+*) button next the *Add TLS Context* link and specify the required details in the *Select TLS Context* window.
+.. If you are creating an endpoint with proxy, accept the base path `/` for the proxy endpoint in the *Path* field, or enter another path. 
 +
-. If you are creating an endpoint with proxy, accept the base path `/` for the proxy endpoint in the *Path* field, or enter another path. 
+For example, if an API is defined with the `Implementation URI` set to `http://jsonplaceholder.typicode.com`, the proxy returns the following URIs based on the value that you configure in the proxy path:
++
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| *Proxy Path* | *Request* | *Response from Proxy*
+|`/` | `https://domain.cloudhub.io/users` | `http://jsonplaceholder.typicode.com/users`
+| `/api` | `https://domain.cloudhub.io/api/users` | `http://jsonplaceholder.typicode.com/users`
+|===
+.. Optionally, expand the *Advanced options* section and specify additional details:
```

Comment:
This is a long step, especially for an optional step; evaluate alternative treatments for this information.

It would also be helpful to explain "optionally," per the style reference entry for "optional steps": under what circumstances ("If you want to") would the audience use this option, or why?

Reply:
Done! Thanks!